aion/web/elm/src/Update.elm

```diff
@@ -89,12 +91,15 @@ update msg model =
                 push_ =
                     Phoenix.Push.init "new:answer" ("rooms:" ++ (toString model.roomId))
                         |> Phoenix.Push.withPayload payload
+                        |> Phoenix.Push.onError (\v -> WrongAnswer)
```

Comment:
Why do we invoke `WrongAnswer` on push? `ReceiveFeedback` is much better choice and it is already prepared to use:
https://github.com/pmrukot/aion/blob/623b9549e929f16f28bdca03d55e32cf83967bfe/aion/web/elm/src/Update.elm#L76

Also it does not feel scalable since we would like to have different types of notifications, not only `onError`. Currently we have three types of feedback message + we would like to have also when user joins the room, leaves the room, scores the point etc.

Details about feedback are in this PR:
https://github.com/pmrukot/aion/pull/48/files#diff-1d980e9fadf285c6fee743d51b4c8e6dR60